en/docs/release-notes.md

```diff
@@ -1,5 +1,24 @@
 # Release Notes
 
+## Jun 23, 2022
+### API updates
+1. Added query parameter 'scope' for GET /report/flow and GET /report/requests.
+2. Updated APIs of service quota to meet the requirements of trial customers.
+3. Added a scheduler to check trial deadline and traffic usage of trial customers, and send out notifications.
+4. Added a scheduler to disable transport service when meets trial deadline or traffic limit.
+5. Reorganized the acceleration regions of existing transports.
+6. Made API query parameters case insensitive for easy use.
+
+### Portal updates
+1. Added a notification module.
+2. Added reminders for trial date and traffic usage when trial customers login in the portal.
+3. Reorganized the acceleration regions.
+4. Display query filters when downloading the billing reports.
+5. Set 'All' as the default value of access domain in Reports page, limited the number of transport and access domain.
```

Comment:
Is the part of the sentence after the comma related to the first part?
I'd replace with  "... Reports page and limited the number of transports and access domains" to improve the English though two parts of the sentence seem to be about different things

Reply:
Replace the sentence with:
"In Reports page, set 'All' as the default value for access domain selection and limited the number of transports and access domains that can be selected."